kong/runloop/balancer.lua

```diff
@@ -1036,6 +1048,43 @@ local function get_upstream_health(upstream_id)
 end
 
 
+--------------------------------------------------------------------------------
+-- Get healthcheck information for a balancer.
+-- @param upstream_id the id of the upstream.
+-- @return table with balancer health info
+local function get_balancer_health(upstream_id)
+
+  local upstream = get_upstream_by_id(upstream_id)
+  if not upstream then
+    return nil, "upstream not found"
+  end
+
+  local using_hc = is_upstream_using_healthcheck(upstream)
+
+  local balancer = balancers[upstream_id]
+  if not balancer then
+    return nil, "balancer not found"
+  end
+
+  local healthchecker
+  local health = "HEALTHCHECKS_OFF"
+  if using_hc then
```

Comment:
it seems the `using_hc` variable is not needed. Replace this with `if is_upstream_using_healthcheck(upstream) then` and remove it from above.

Reply:
Good catch, thanks. Done!